src/components/MultiLink/MultiLink.vue

```diff
@@ -197,7 +198,9 @@ export default Vue.extend({
       nodes.forEach((node) => {
         // If the position is not defined for x or y, generate it
         if (node.x === undefined || node.y === undefined) {
+          // eslint-disable-next-line no-param-reassign
```

Comment:
Are you certain this code triggers this error? Manipulating the elements of an array using `forEach()` is such a common pattern that I'm surprised TS doesn't allow it by default, especially when you're not, strictly speaking, reassigning the parameter here, you're mutating it by assigning to one of its members.

Reply:
Positive, here's the error:

`error  Assignment to property of function parameter 'node'  no-param-reassign`

Reply:
Huh. Ok then. Carry on :cat: